Tests/ApolloTests/HTTPTransportTests.swift

```diff
@@ -190,6 +190,65 @@ class HTTPTransportTests: XCTestCase {
     self.wait(for: [expectation], timeout: 10)
   }
   
+  func testRetryDelegateReturnsCorrectError() throws {
```

Comment:
I think this should probably be split into two tests: One that tests the scenario where you want the returned error and one that tests the scenario where you want the retry error.

Reply:
There is "retry" test already (StarWarsServerTests:28, HTTPTransportTests:412). The only test that was needed is returning correct error.

Reply:
Ah, I phrased that poorly: One that tests the scenario where you want the original error received from the first call to the server returned, and one that tests the scenario where you want the error that was returned during the login instead. Does that make more sense?